Source/driver/Castro_io.cpp

```diff
@@ -1114,6 +1132,14 @@ Castro::plotFileOutput(const std::string& dir,
     }
 #endif
 
+#ifdef REACTIONS
+#ifndef TRUE_SDC
+    if (store_burn_weights) {
+        MultiFab::Copy(plotMF, getLevel(level).burn_weights, 0, cnt, Castro::burn_weight_names.size(),0);
```

Comment:
Can you increment `cnt` here appropriately so that we don't make a mistake if we add something later?

Reply:
done